tests/datadriven/__init__.py

```diff
@@ -42,7 +42,8 @@ def makeTests(testDataDir, testClass, fnmatchGlob="*"):
     """
     for localId in os.listdir(testDataDir):
         if (fnmatch.fnmatch(localId, fnmatchGlob) and
-                not fnmatch.fnmatch(localId, '*.json')):
+                not fnmatch.fnmatch(localId, '*.json') and
+                not localId[0] == "."):  # hidden files start with `.`
```

Comment:
``not localId.startswith(".")`` a bit more reabable perhaps? Or at least ``localId[0] != "."``?